s3_website.yml

```diff
@@ -43,7 +43,6 @@ cloudfront_distribution_config:
 # concurrency_level: 5
 
 redirects:
-  articles/can-multi-year-ssl-certificate: /articles/can-multi-year-ssl-certificates/
   articles/secondary-dns-support: /articles/secondary-dns/
```

Comment:
The second one works:

http://support.dnsimple.com/articles/secondary-dns-support

=>

http://support.dnsimple.com/articles/secondary-dns/